### ISSUE TITLE
Update OR remove (allowing any) the allowlist of transformers available in manifests.transformers used via Kpt

### ISSUE THREAD

**Issue by hampus77** [GoogleContainerTools/skaffold#8189]
Hi,
What´s the reasoning behind the limited set of transformers available in manifests.transformers? The documentation states "For a list of kpt supported containers to use in the manifests.transform schema" and links to https://catalog.kpt.dev/ which kind of gives you the impression that all functions listed there should be available. However, only 5 of them are actually allowed.

I guess there´s a reason for this but it´s unclear

**Comment by renzodavid9** [comment]
Hi @hampus77. Thanks for your question. I see what you are saying in terms of discrepancy between what could be understood from the documentation, and what ends up happening using Skaffold, but I'll double check with the team first. In terms of the reason why, I'll need to dig more before coming up with more info. Thanks!

**Comment by yuwenma** [comment]
The catalog.kpt.dev is a live list where we continue developing and adding curated Kpt and Google cloud Kpt functions to it. When we generated the Skaffold fn whitelist, there are only 5 curated functions suitable for Skaffold. 
Based on the current list, I think the other suitable functions are `set-namespace`, `set-annotations`, `set-labels`, `starlark`, `apply-replacements`, `set-image`.

@aaron-prindle @tejal29

**Comment by aaron-prindle** [comment]
@hampus77 from the information from @yuwenma it seems that the allow/deny list should at the very least be updated to reflect t

TLDR; there is no reason for the subset currently - the Skaffold team needs to update the allow/deny list to include all relevant functions or what might also make sense is removing the allow/deny list entirely

**Comment by aaron-prindle** [comment]
Related: https://github.com/GoogleContainerTools/skaffold/issues/6784
